app/controllers/orders_controller.rb

```diff
@@ -2,15 +2,19 @@
 
 class OrdersController < ApplicationController
   def index
-    @orders = Order.where(employee: current_employee)
+    if params[:status].present?
+      render :index, locals: { orders: Order.where(employee: current_employee).filter_by_status(params[:status]) }
+    else
+      render :index, locals: { orders: Order.where(employee: current_employee) }
```

Comment:
I would consider moving rendering outside if-else and passing default filtering instead

Reply:
let's leave it like it is right now - we will refactor it later, when more sorting options will be available - then the reason for refactor will be easier to understand